docs/docsite/rst/shared_snippets/galaxy_server_list.txt

```diff
@@ -1,11 +1,19 @@
 
 
+If you use the ``ansible-galaxy`` command, you need to first configure the server lists that command communicates with. You can configure this as follows:
```

Comment:
You don't need to do this if using the `https://galaxy.ansible.com/`, only really for things like Automation Hub or dev/custom Galaxy instances. That's why I had the link to `:ref:galaxy_server` as that's the default server to use.